spec/models/instance_spec.rb

```diff
@@ -1,6 +1,29 @@
 require 'rails_helper'
 
 RSpec.describe Instance, type: :model do
+  let(:instance) { build(:instance) }
+
+  context 'when host format is invalid' do
+    it "is invalid" do
+      hosts = %w[example,com www_.example.org example_.org example.org_
+                      example..org example.org.]
+      hosts.each do |invalid_host|
+        instance.host = invalid_host
+        expect(instance).not_to be_valid
+      end
+    end
+  end
+
+  context 'when host format is valid' do
+    it "is valid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.